### PR TITLE
Bootstrap application linting is broken - Closes #6376

### LINKED ISSUE #6376: Bootstrap application linting is broken
### Expected behavior

After we bootstrap the application `lisk init` and then run the `npm run lint`, the linting should work fine. 

### Actual behavior

Linting at the bootstrap application is broken

```
> npm run lint

> lisk-name-service@0.1.0 lint /my/app
> eslint --ext .js,.ts .


Oops! Something went wrong! :(

ESLint: 7.22.0

Error: Cannot read config file: /my/app/node_modules/eslint-config-prettier/@typescript-eslint.js
Error: "prettier/@typescript-eslint" has been merged into "prettier" in eslint-config-prettier 8.0.0. See: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21
Referenced from: /my/app/.eslintrc.js
    at Object.<anonymous> (/my/app/node_modules/eslint-config-prettier/@typescript-eslint.js:1:69)
    at Module._compile (/my/app/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at module.exports (/my/app/node_modules/import-fresh/index.js:32:59)
    at loadJSConfigFile (/my/app/node_modules/@eslint/eslintrc/lib/config-array-factory.js:225:16)
    at loadConfigFile (/my/app/node_modules/@eslint/eslintrc/lib/config-array-factory.js:309:20)
    at ConfigArrayFactory._loadConfigData (/my/app/node_modules/@eslint/eslintrc/lib/config-array-factory.js:609:42)
```

### Steps to reproduce

```
lisk init /path/to/app
cd /path/to/app
npm run lint
```

### Which version(s) does this affect? (Environment, OS, etc...)
development

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?

This PR resolves #6376

### How was it solved?

- Fix eslint config for bootstrap application

### How was it tested?

- Tested manually

### CODE REVIEW

File: commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { getApplication } from '../../app/app';
 import { registerModules } from '../../app/modules';
 
 export class GenesisBlockCommand extends BaseGenesisBlockCommand {
-		public getApplication(
+	// eslint-disable-next-line class-methods-use-this
```
Comment (shuse2) [inline commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts:7]: I think this rule was removed in the lisk-base. Are you sure this has error?

Comment (nazarhussain) [reply commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts:7]: Its not an error, its causing warning.

Comment (shuse2) [reply commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts:7]: Is it using correct lisk-base version? https://github.com/LiskHQ/eslint-config-lisk-base/blob/master/ts.js#L15
it looks it should be off

Comment (nazarhussain) [reply commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts:7]: Yes the version itself is correct. 

```
		"eslint-config-lisk-base": "2.0.0",
```

Comment (nazarhussain) [reply commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts:7]: @shuse2 Found the issue, fixing it now. We were using wrong eslint config.

**Review by shuse2 — APPROVED** [review]

**Review by Incede — APPROVED** [review]

**Pull request merged**